connector_equipment_service/models/maintenance_equipment.py

```diff
@@ -0,0 +1,29 @@
+# Copyright (C) 2019 Open source Integrators
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, models
+
+
+class MaintenanceEquipment(models.Model):
+    _inherit = 'maintenance.equipment'
+    _description = 'Maintenance Equipment'
+
+    @api.multi
+    def add_service(self, service_profile):
+        return True
```

Comment:
How about returning some "Not Implemented" exception?

Reply:
@dreispt The Design document doesn't specify any "Not Implemented" exceptions, I do not believe they are necessary.

Here is the design doc: https://wiki.opensourceintegrators.com/amp/index.php?title=Develop_calls_to_Amplex_API_based_on_scheduled_action_results_to_enable_or_cancel_service